dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcResult.java

```diff
@@ -109,4 +92,32 @@ public boolean hasException() {
     public String toString() {
         return "RpcResult [result=" + result + ", exception=" + exception + "]";
     }
+
+    /**
+     * fix issue#619
```

Comment:
Can you provide some more valuable docs?

Reply:
this pr is just optimization for #2956 , and the docs for #2956 : https://www.yuque.com/fa902k/id5z6r/sr041v

Reply:
Sorry, there is a problem with my expression, I mean a comment like this:

```
// Process string to char array for generic invoke
// See
// - https://github.com/apache/incubator-dubbo/issues/2003
```

Make it easier for developers to know the reason for this change.

Reply:
@carryxyh Any suggestions to my following docs?

Reply:
Hi, xiang
I mean a java comment like this:

```
    /**
     * we need deal the stack trace to avoid NullPointerException
     * <p>
     * see https://github.com/apache/incubator-dubbo/pull/2956
     * and https://github.com/apache/incubator-dubbo/issues/619
     *
     * @param e exception
     * @return exception after deal with stack trace
     */
```

What do u think?

Reply:
ok